packages/editor-ui/src/components/ValueSurvey.vue

```diff
@@ -72,67 +88,118 @@ export default mixins(workflowHelpers).extend({
+			this.loading = true;
+			this.$store.dispatch('settings/submitValueSurvey', { value: this.form.value }).then((response: IN8nPromptResponse) => {
+				if (response.updated) {
+					this.$telemetry.track('User responded value survey score', {
+						instance_id: this.$store.getters.instanceId,
+						nps: this.form.value,
+					});
+				}
+				this.loading = false;
+			});
+
 		},
 		send(): void {
-			if (this.isEmailValid) {
-				this.$store.dispatch('settings/submitValueSurvey', {email: this.form.email, value: this.form.value});
-				this.$telemetry.track('User responded value survey email', { instance_id: this.$store.getters.instanceId, email: this.form.email });
-				this.$showMessage({
-					title: 'Thanks for your feedback',
-					message: `If you’d like to help even more, answer this <a target="_blank" href="https://n8n-community.typeform.com/quicksurvey#how_disappointed=${this.form.value}&instance_id=${this.$store.getters.instanceId}">quick survey.</a>`,
-					type: 'success',
-					duration: 15000,
-				});
-				this.$store.commit('ui/closeTopModal');
-			}
+			this.$store.dispatch('settings/submitValueSurvey', {
+				email: this.form.email,
+				value: this.form.value,
+			}).then((response: IN8nPromptResponse) => {
+				if (response.updated) {
+					this.$telemetry.track('User responded value survey email', {
+						instance_id: this.$store.getters.instanceId,
+						email: this.form.email,
+					});
+					this.$showMessage({
+						title: 'Thanks for your feedback',
+						message: `If you’d like to help even more, answer this <a target="_blank" href="https://n8n-community.typeform.com/quicksurvey#nps=${this.form.value}&instance_id=${this.$store.getters.instanceId}">quick survey.</a>`,
+						type: 'success',
+						duration: 15000,
+					});
+					this.$store.commit('ui/closeTopModal');
+				}
+			});
+
 		},
 	},
 	mounted() {
-		this.$telemetry.track('User shown value survey', { instance_id: this.$store.getters.instanceId});
+		this.$telemetry.track('User shown value survey', {
+			instance_id: this.$store.getters.instanceId,
+		});
 	},
 });
 </script>
 
 <style module lang="scss">
 .title {
 	height: 16px;
-	font-size: var(--font-size-m);
-	line-height: 18px;
 	text-align: center;
-	color: var(--color-text-xlight);
 }
 
 .content {
 	display: flex;
 	justify-content: center;
 
-	.buttons {
-		button {
-			margin: 0 8px;
-			background-color: var(--color-background-xlight);
-			border: var(--color-background-xlight);
-			border-radius: 4px;
-			color: var(--color-background-dark);
-			font-size: var(--font-size-s);
+	.wrapper {
+		display: flex;
+		flex-direction: column;
+
+		.buttons {
+			display: flex;
+
+			.button {
+				margin: 0 8px;
+
+				button {
+					width: 28px;
+					border: var(--color-background-xlight);
+					border-radius: 4px;
+					color: var(--color-background-dark);
+					font-size: var(--font-size-s);
+					font-weight: var(--font-weight-bold);
+				}
```

Comment:
not sure why any of this is needed? we should not have to override components.. 
fyi there's a text button element https://n8n-io.github.io/n8n-storybook/?path=/story/atoms-button--text

Reply:
in this case, might be better to create a new component rather than override

Reply:
@mutdmour I completely understand that I have not to override the components, but the N8N-button do not give me a chance to change the font size, font-weight, color, border-radius etc etc as is done in design. I will create new component instead of this.

Reply:
Simplified. I added simple button instead of using n8n-button

Reply:
@mutdmour Updated. Created N8N-Square-Button